maestro-xcuitest-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -19,6 +19,7 @@ import java.util.concurrent.TimeUnit
 class XCTestDriverClient(
     private val host: String = "localhost",
     private val port: Int = 22087,
+    private val isStudio: Boolean,
```

Comment:
Let's not couple XCTestDriverClient to studio (semantically). Can you find a better name for this property?

Reply:
Done renamed it! ✅